app/_redirects

```diff
@@ -34,3 +34,33 @@ https://prometheus.metrics.kuma.io/port /docs/LATEST_RELEASE/reference/kubernete
 https://prometheus.metrics.kuma.io/path /docs/LATEST_RELEASE/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-path/ 302
 https://traffic.kuma.io/exclude-inbound-ports /docs/LATEST_RELEASE/reference/kubernetes-annotations/#traffic-kuma-io-exclude-inbound-ports/ 302
 https://traffic.kuma.io/exclude-outbound-ports /docs/LATEST_RELEASE/reference/kubernetes-annotations/#traffic-kuma-io-exclude-outbound-ports/ 302
+
+# kuma IA redirects
+/docs/:version/introduction/deployments/  /docs/:version/production/deployment/
+/docs/:version/deployments/stand-alone/ /docs/:version/production/deployment/stand-alone/
+/docs/:version/deployments/multi-zone/  /docs/:version/production/deployment/multi-zone/
+/docs/:version/deployments/multi-zone/#usage  /docs/:version/production/cp-deployment/multi-zone/
+/docs/2.1.x/deployments/multi-zone/#usage /docs/2.1.x/production/cp-deployment/multi-zone/
+/docs/2.1.x/deployments/stand-alone/#usage  /docs/2.1.x/production/cp-deployment/stand-alone/
```

Comment:
@fabianrbz These ones still aren't working, but because I split the page, I think the "2.1.x" has to be in the URL since 2.1.x should be the only impacted version.